server/pom.xml

```diff
@@ -1,153 +1,185 @@
+            <id>create-archive</id>
+            <goals>
+              <goal>single</goal>
+            </goals>
+            <phase>package</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>1.6.0</version>
+        <configuration>
+          <mainClass>${main.class}</mainClass>
+        </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>java</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>com.coveo</groupId>
+        <artifactId>fmt-maven-plugin</artifactId>
+        <version>2.9</version>
+        <configuration>
+          <displayFiles>true</displayFiles>
+          <verbose>true</verbose>
+          <filesNamePattern>.*\.java</filesNamePattern>
+          <skip>false</skip>
+          <skipSortingImports>false</skipSortingImports>
+          <style>google</style>
+        </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>format</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M3</version>
+      </plugin>
+      <plugin>
+        <groupId>com.github.ekryd.sortpom</groupId>
+        <artifactId>sortpom-maven-plugin</artifactId>
+        <version>2.8.0</version>
+        <configuration>
+          <createBackupFile>false</createBackupFile>
+          <predefinedSortOrder>custom_1</predefinedSortOrder>
+          <lineSeparator>\n</lineSeparator>
+          <encoding>${project.build.sourceEncoding}</encoding>
+          <sortProperties>true</sortProperties>
+          <keepBlankLines>true</keepBlankLines>
+          <sortDependencies>scope</sortDependencies>
+        </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>sort</goal>
+            </goals>
+            <phase>verify</phase>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+  <url>http://maven.apache.org</url>
+  <developers>
+    <developer>
+      <name>Dustin Roan</name>
+      <email>dustin.a.roan@gmail.com</email>
+      <organization>ShowMeTheRoapes</organization>
+      <roles>
+        <role>Software Developer</role>
+      </roles>
+    </developer>
+    <developer>
+      <name>Alex Hoopes</name>
+      <email>kevin.alex.hoopes@gmail.com</email>
+      <organization>ShowMeTheRoapes</organization>
+      <roles>
+        <role>Software Developer</role>
+      </roles>
+    </developer>
+  </developers>
+  <issueManagement>
+    <system>Github Issues</system>
+    <url>https://github.com/ShowMeTheRoapes/Paracord/issues</url>
+  </issueManagement>
```

Comment:
Holy cow did you add some of this manually? 😲

Reply:
It kind of hates xml files, but yes I have added everything on this page :)

Reply:
Ewwww lol but impressive sir :D